src/lib/utils.ts

```diff
@@ -4,6 +4,16 @@ import { logger } from './logger'
 import { Revision } from './models'
 import { realtime } from './realtime'
 
+/*
+Converts a map from string to something into a plain JSO for transmitting via a websocket
```

Comment:
Can we write it out? JSO was mistakenly considered a typo first, should be JS object.

Reply:
done